janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1810,7 +1811,22 @@ private void testCollection(Cardinality cardinality, String property, Object val
             assertEquals(v1, getOnlyElement(graph.query().has(property, Geo.WITHIN, Geoshape.circle(2.0, 2.0, 0.1)).vertices()));
         }
 
-
+        
+        // Test traversal property drop Issue #408
```

Comment:
Should this be a separate test method to isolate it from other tests? Seems like it's independent from what's above.